### PR TITLE
Fix broken links in Pipelines

### PR DESCRIPTION
**Pull request by hbelmiro**
Resolves https://github.com/kubeflow/website/issues/3808.

### CODE REVIEW

**Comment by hbelmiro** [comment]
@thesuperzapper you may want to take a look at this PR.

**Comment by hbelmiro** [comment]
@terrytangyuan can you please take a look at this?

**Comment by hbelmiro** [comment]
@chensun can you please take a look?

File: content/en/docs/components/pipelines/legacy-v1/installation/upgrade.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ For upgrade instructions, refer to distribution specific documentations:
 
     For detailed configuration and migration instructions for both options, refer to [Argo Workflow Executors](/docs/components/pipelines/legacy-v1/installation/choose-executor/).
 
-* **Notice**: [Kubeflow Pipelines SDK v2 compatibility mode](/docs/components/pipelines/legacy-v1/sdk/v2-compatibility/) (Beta) was recently released. The new mode adds support for tracking pipeline runs and artifacts using ML Metadata. In v1.7 backend, complete UI support and caching capabilities for v2 compatibility mode are newly added. We welcome any [feedback](https://github.com/kubeflow/pipelines/issues/6451) on positive experiences or issues you encounter.
```
Comment (zijianjoy) [inline content/en/docs/components/pipelines/legacy-v1/installation/upgrade.md:37]: This is a release note, so it shouldn't be changed for its use of `SDK v2 compatibility mode`.  We can remove the link if that is blocking the link check.

Comment (hbelmiro) [reply content/en/docs/components/pipelines/legacy-v1/installation/upgrade.md:37]: Yeah, makes sense. Removed.

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you @hbelmiro for the contribution!

**Pull request merged**